examples/hacker_news_pwa/web/index.html

```diff
@@ -112,7 +107,7 @@
         <a href="https://webdev.dartlang.org/angular">
           <img src="/images/logo.svg" alt="AngularDart logo" />
         </a>
-        <a href="">top</a>
+        <a href="" class="active">top</a>
```

Comment:
This shouldn't be here.

1. You moved the style from this file, to app_component.css. By the time that file is loaded this static HTML will have been replaced anyways.
2. Even if you moved the style back, we don't want to assume users loaded the app at /. They might have loaded /ask, so highlighting this link preemptively would be incorrect.